app/assets/javascripts/admin/views/statistic/statistic_view.js

```diff
@@ -1,57 +1,213 @@
+                    y: {
+                      label: "hours / week",
+                    },
+                  }}
+                  legend={{
+                    show: false,
+                  }}
+                />
+              </Spin>
+            </Card>
+          </Col>
+          <Col span={8}>
+            <Card title="Achievements">
+              <Spin spinning={this.state.isAchievementsLoading} size="large">
+                <table className="table">
+                  <tbody>
+                    <tr>
+                      <td>Number of Users</td>
+                      <td>
+                        {this.state.achievements.numberOfUsers}
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Number of Datasets</td>
+                      <td>
+                        {this.state.achievements.numberOfDatasets}
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Number of Annotations</td>
+                      <td>
+                        {this.state.achievements.numberOfAnnotations}
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Number of Trees</td>
+                      <td>
+                        {this.state.achievements.numberOfTrees}
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Number of Open Assignments</td>
+                      <td>
+                        {this.state.achievements.numberOfOpenAssignments}
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </Spin>
+            </Card>
+          </Col>
+        </Row>
+        <Row gutter={16}>
+          <Col span={24}>
+            <Card
+              title={`Best Tracers for Week ${this.state.startDate.format(
+                "DD.MM",
+              )} - ${this.state.endDate.format("DD.MM.YYYY")}`}
+              style={{ marginTop: 30, marginBotton: 30 }}
+            >
+              <Spin spinning={this.state.isTimeEntriesLoading} size="large">
+                <Table
+                  dataSource={this.state.timeEntries}
+                  rowKey={entry => entry.user.id}
+                  style={{ marginTop: 30, marginBotton: 30 }}
+                  pagination={false}
+                >
+                  <Column
+                    title="User"
+                    dataIndex="user"
+                    key="user"
+                    render={user => `${user.firstName} ${user.lastName} (${user.email})`}
+                  />
+                  <Column
+                    title="Duration"
+                    dataIndex="tracingTimes"
+                    key="tracingTimes"
+                    render={tracingTimes =>
+                      tracingTimes.map(timeEntry => {
+                        const minutes = timeEntry.tracingTime / 1000 / 60;
```

Comment:
Doesn't Momentjs offer some human readable utility function?

Reply:
Unfortunately there is no good way in moment.js. The time entry is a duration which requires `moment.duration`. However, durations don't support formatiing :-( There is a bunch of hacky solution out there, but at that point we can just do the simple math.

I have been following this thread for a year now without any progress:                         https://github.com/moment/moment/issues/1048